requirements.txt

```
@@ -1,4 +1,4 @@
 # Development requirements
-spacy>=4.0.0.dev0
+spacy>=3.1.0
```

We don't need to revert this, right? The branch is still the `v4` branch.
```suggestion
spacy>=4.0.0.dev0
```

---

Ah, yes, thanks, fixed!